admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -477,6 +478,16 @@ class AdminModuleController {
     let newValue;
     let defaultMax;
 
+    const paramsUrl = (new URL(document.location)).searchParams;
+    const findModule = paramsUrl.get('find');
+
+    if (findModule !== null && self.findModuleUsed !== true) {
```

Comment:
```suggestion
    if (findModule && self.findModuleUsed !== true) {
```

Isn't it enough? I barely use `myVar !== null` so it's pretty surprising

Reply:
Also findModule could be `undefined` and so it would jump into the condition